zkevm-circuits/src/evm_circuit.rs

```diff
@@ -1,1257 +1,284 @@
-                            meta.query_fixed(*column, Rotation::cur()),
-                        )
-                    })
-                    .collect::<Vec<_>>()
-            });
-        }
-
-        // Configure byte code lookups
-        for bytecode_lookup in bytecode_lookups.iter() {
-            meta.lookup_any(|meta| {
-                bytecode_lookup
-                    .iter()
-                    .zip(bytecode_table.iter())
-                    .map(|(expr, column)| {
-                        (
-                            expr.clone(),
-                            meta.query_advice(*column, Rotation::cur()),
-                        )
-                    })
-                    .collect::<Vec<_>>()
-            });
-        }
-        // Configure rw lookups
-        for rw_lookup in rw_lookups.iter() {
-            meta.lookup_any(|meta| {
-                rw_lookup
-                    .iter()
-                    .zip(rw_table.iter())
-                    .map(|(expr, column)| {
-                        (
-                            expr.clone(),
-                            meta.query_advice(*column, Rotation::cur()),
-                        )
-                    })
-                    .collect::<Vec<_>>()
-            });
+            execution,
         }
     }
 
-    fn load_fixed_tables(
+    /// Load fixed table
+    pub fn load_fixed_table(
         &self,
         layouter: &mut impl Layouter<F>,
-        including_large_tables: bool,
+        fixed_table_tags: Vec<FixedTableTag>,
     ) -> Result<(), Error> {
         layouter.assign_region(
             || "fixed table",
             |mut region| {
-                let mut offset = 0;
-
-                // Noop
-                for (idx, column) in self.fixed_table.iter().enumerate() {
-                    region.assign_fixed(
-                        || format!("Noop: {}", idx),
-                        *column,
-                        offset,
-                        || Ok(F::zero()),
-                    )?;
-                }
-                offset += 1;
-
-                // Range256
-                for idx in 0..256 {
-                    region.assign_fixed(
-                        || "Range256: tag",
-                        self.fixed_table[0],
-                        offset,
-                        || Ok(F::from(FixedLookup::Range256 as u64)),
-                    )?;
-                    region.assign_fixed(
-                        || "Range256: value",
-                        self.fixed_table[1],
-                        offset,
-                        || Ok(F::from(idx as u64)),
-                    )?;
-                    for (idx, column) in
-                        self.fixed_table[2..].iter().enumerate()
-                    {
+                for (offset, row) in std::iter::once([
+                    F::zero(),
+                    F::zero(),
+                    F::zero(),
+                    F::zero(),
+                ])
```

Comment:
```suggestion
                for (offset, row) in std::iter::once([F::zero(); 4])
```

Reply:
Thanks, fixed in [`13efaecd`](https://github.com/appliedzkp/zkevm-circuits/pull/196/commits/13efaecd5548c6c15b7f60221368d5954421a6df).